scripts/StateJsonConvert/README.md

```diff
@@ -0,0 +1,58 @@
+# Convert State JSON from Zilliqa API Format to Scilla Format
+
+The [Zilliqa API]((https://dev.zilliqa.com/docs/apis/api-contract-get-smartcontract-state))
+to fetch the state of a contract produces a JSON in the format
+
+```
+{
+  "_balance": "0",
+  "admins": {
+    "0xdfa89866ae86632b36361d53b76c1373448c28fa": {
+      "argtypes": [],
+      "arguments": [],
+      "constructor": "True"
+    }
+  }
+}
+```
+
+This program converts it into a format that Scilla understands. Since
+there is some loss of information in the API output (the type of a field,
+for example), this program also requires the `-contractinfo` output of
+scilla-checker as an additional input to find this missing information.
```

Comment:
You'll need to provide the init file to scilla-checker as well. Otherwise, scilla-checker will output the wrong name for ADTs.

Reply:
Will mention that.